app/bundles/CoreBundle/Entity/Notification.php

```diff
@@ -16,7 +16,7 @@ class Notification
     protected $id;
 
     /**
-     * @var User|null
+     * @var User
```

Comment:
Used in Line 71 as part of `createManyToOne`

Reply:
The entities are not created only by Doctrine from the database. But also by
```php
$notification = new Notification();
```
And when you call `$notification->getUser()` then it will be `null`.